apps/src/sites/studio/pages/levels-standaloneVideo-main.js

```diff
@@ -0,0 +1 @@
+/* This file isn't used. */
```

Comment:
Could this file be deleted? Or does it need to exist for webpack?

Reply:
Yes, our build process looks for this file, and I didn't want to make changes to that system in this PR if possible. :)

Reply:
Ah okay, out of curiosity would removing `standaloneVideo` from the list of entrypoints in Gruntfile/checkEntryPoints skip the check for this file?

Reply:
I had tried a bunch of things and thought it necessary, but it seems okay without the entry and the file, now.